crates/oct-ctl/src/main.rs

```diff
@@ -12,17 +14,79 @@ async fn index() -> impl Responder {
         .output();
 
     match command {
-        Ok(_) => "Success",
-        Err(_) => "Error",
+        Ok(res) => {
+            println!("Result: {}", String::from_utf8_lossy(&res.stdout));
+            "Success"
+        }
+        Err(err) => {
+            println!("{}", err);
+            "Error"
+        }
+    }
+}
+
+const MAX_SIZE: usize = 262_144; // max payload size is 256k
```

Comment:
`256 * 1024` is more readable. And if you rename `MAX_SIZE` to `MAX_SIZE_BYTES`, it'll be even more readable